hudi-spark/src/test/java/HoodieJavaApp.java

```diff
@@ -104,6 +104,8 @@ public void run() throws Exception {
     SparkSession spark = SparkSession.builder().appName("Hoodie Spark APP")
         .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer").master("local[1]").getOrCreate();
     JavaSparkContext jssc = new JavaSparkContext(spark.sparkContext());
+    spark.sparkContext().setLogLevel("WARN");
+    jssc.setLogLevel("WARN");
```

Comment:
we need both?

Reply:
I guess just the first one is fine. Will remove the second line.